pkg/ui/styl/components/select.styl

```diff
@@ -15,3 +17,13 @@
     display inline-block
     vertical-align middle
     width 200px
+
```

Comment:
These are not shameful, and they are in the "right place" as such; however, you should probably just move them to the shame file anyway.  That said, please put a comment here that "style overrides for react-select are in shame.styl due to excessive use of !important"